### DISCUSSION TITLE
Fruit is not rendered properly in argon skin

### DISCUSSION THREAD

**Discussion by rafroad** [ppy/osu#21033] [Q&A] [answered]
Was testing out the new CTB argon skin on mobile and it is not rendered properly ![image](https://user-images.githubusercontent.com/56687086/199035737-e5a6bc76-67d6-4884-af2c-990f00d7dbdc.png)
![image](https://user-images.githubusercontent.com/56687086/199035784-c088a8c3-dcea-431a-923a-377c4df7b27b.png)
Pic for reference the map is snow halation namki insane

**Comment by salmonslay** [comment]
For me they're rendered, however they're circles and not blobs (Android 12)

![image](https://user-images.githubusercontent.com/47401343/199065739-cb7bbd8d-9d83-4a4a-a522-492de976f1f4.png)

**Reply by nagi-desuuu** [reply]
Another similar anomaly can be observed on a device running Android 9, although it would be best described as a split-up circle with different scaling sizes applied on each of the layers.
(I'm sorry for sounding a little too fancy, but hopefully the inline image might help.)

![osu_2022-11-01_03-21-34.jpg](https://user-images.githubusercontent.com/75596775/199092517-648ee1c3-25dd-44a4-b40e-975757f67410.jpg)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu-framework/pull/5495
